src/pages/VouchingPage/NomineeCard.tsx

```diff
@@ -128,14 +129,21 @@ const TextOnlyTooltip = withStyles({
   },
 })(Tooltip);
 
-export const NomineeCard = ({ nominee }: { nominee: INominee }) => {
+export const NomineeCard = ({
+  nominee,
+  refetchNominees,
+}: {
+  nominee: IActiveNominee;
+  refetchNominees: () => void;
+}) => {
   const classes = useStyles();
   const { vouchUser } = useApiWithSelectedCircle();
   const { circle, myUser } = useSelectedCircle();
+  const [vouching, setVouching] = useState(false);
   const vouchDisabled =
     myUser && circle
       ? nominee.nominated_by_user_id === myUser.id ||
-        nominee.nominations.some(user => user.id === myUser.id) ||
+        nominee.nominations.some(voucher => voucher.voucher_id === myUser.id) ||
```

Comment:
```suggestion
        nominee.nominations.some(n => n.voucher_id === myUser.id) ||
```

i know this is confusing because the relationship is called "nominations" but the database table is called "vouches"